codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Headers.java

```diff
@@ -114,6 +113,20 @@ public Http2Headers clear() {
     }
 
     @Override
+    public boolean equals(Object o) {
```

Comment:
Yes sir it was :(

We were previously using all `AsciiString` in tests apparently so the value comparisons using POJO `equals` was sufficient.